src/components/tag/tag.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div :class="`apos-tag apos-tag--${modifier}`" :disabled="disabled">
```

Comment:
should this whole ditty be a button? or do you just click the `X` to remove the tag?

Reply:
checking out some other web apps, I like the pattern where only the icon is clickable (in the case of Remove, anyway).

Wrapping the icon component in a `<button>` tag..